apps/condo/domains/billing/schema/BillingProperty.js

```diff
@@ -73,7 +68,7 @@ const BillingProperty = new GQLListSchema('BillingProperty', {
             },
         },
     },
-    plugins: [uuided(), versioned(), tracked(), softDeleted(), dvAndSender(), historical()],
+    plugins: [uuided(), addressService({ fieldsHooks: {}, resolveAddressFields: ({ addressFields }) => addressFields }), versioned(), tracked(), softDeleted(), dvAndSender(), historical()],
```

Comment:
Why not add a default value to the `addressService()` function signature? Then you can call without copying all parameters.
```suggestion
    plugins: [uuided(), addressService(), versioned(), tracked(), softDeleted(), dvAndSender(), historical()],
```

Reply:
addressService() are throwing exception:
TypeError: Cannot read properties of undefined (reading 'fieldsHooks')

Reply:
Yes, that is the reason why I've suggested adding a default value to `addressService` function signature.
const addressService = ({fieldHooks={}...} **_= {}_**) => {...}